projects/batfish-common-protocol/src/main/java/org/batfish/datamodel/pojo/TestrigQuestion.java

```diff
@@ -0,0 +1,85 @@
+package org.batfish.datamodel.pojo;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.MoreObjects;
+import java.util.Objects;
+import org.batfish.datamodel.answers.Answer;
+
+/**
+ * The {@link TestrigQuestion TestrigQuestion} is an Object representation of the testrig question
+ * for BatFish service.
+ *
+ * <p>Each {@link TestrigQuestion TestrigQuestion} contains the name of the question, the question
+ * content, an environment, and the answer of the question in that environment.
+ */
+public class TestrigQuestion {
```

Comment:
Why is this not just `Question`? What is `Testrig` about this?

Why does it have an `Answer` field?

Reply:
Questions under testrig folder are created by commands like 'get' or 'answer'(need to load question json first) command, when we create the question, we also generate the answer for that question, so I put the answer as part of TestrigQuestion. Also, I named it TestrigQuestion because it is different from Questions under analysis, which only contains Question instance, does not requires the questions been answered.